internal/pkg/consensus/protocol.go

```diff
@@ -29,7 +29,7 @@ import (
 type Protocol interface {
 	// RunStateTransition returns the state root CID resulting from applying the input ts to the
 	// prior `stateID`.  It returns an error if the transition is invalid.
-	RunStateTransition(ctx context.Context, ts block.TipSet, blsMsgs [][]*types.UnsignedMessage, secpMsgs [][]*types.SignedMessage, ancestors []block.TipSet, parentWeight fbig.Int, parentStateRoot cid.Cid, parentReceiptRoot cid.Cid) (cid.Cid, []vm.MessageReceipt, error)
```

Comment:
FYI I've been meaning to delete this forever.  In the future you can get rid of this instead of fixing.

Reply:
I had a quick go. Some re-wiring is needed to get the `BlockTime()` to the right place, so left for later.